### ISSUE TITLE
Ping Spiking with FPS Unlocker

### ISSUE THREAD

**Issue by xephiana** [pizzaboxer/bloxstrap#816]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

When I unlock my FPS with bloxstrap, my ping seems to spike to 300-1000+, when I set the value to 0 and it recaps at 60 it's just fine. Any way to fix this?

**Comment by pizzaboxer** [comment]
Nope, that's just a problem with Roblox. If it's affecting you that severely, you'll have no choice other than to not use it.

**Comment by BokBoy** [comment]
Hey, I'm getting a similar problem.
When I uncap my fps, I don't get my full 144fps that I'd normally get. I tried different renderers and it doesn't give me back my 500+fps that I normally get with older versions of bloxstrap.
Would that also be a Roblox problem or a Bloxstrap Problem??
Thanks.

**Comment by CodingWithAnxiety** [comment]
> Hey, I'm getting a similar problem. When I uncap my fps, I don't get my full 144fps that I'd normally get. I tried different renderers and it doesn't give me back my 500+fps that I normally get with older versions of bloxstrap. Would that also be a Roblox problem or a Bloxstrap Problem?? Thanks.
> 
> Edit System information Ryzen 7 3700X 16gb ram Dual Channel 8gb Radeon RX 5700 XT
> 
> Expecting a min of 144hz as main display is 144hz, GPU can get more

Not an issue with bloxstrap or the FPS unlocker, but roblox in-of-itself.

**Comment by CodingWithAnxiety** [comment]
> Nope, that's just a problem with Roblox. If it's affecting you that severely, you'll have no choice other than to not use it. Keeping this as an informational post until I put a proper notice for this in the menu.

I'm doing some work today and could go ahead and add it to the menu if you'd like.

**Comment by pizzaboxer** [comment]
> When I uncap my fps, I don't get my full 144fps that I'd normally get. 

How is this remotely the same problem?

**Comment by dooory** [comment]
> ### Acknowledgement of preliminary instructions
> * [x]  I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
> 
> ### What problem did you encounter?
> When I unlock my FPS with bloxstrap, my ping seems to spike to 300-1000+, when I set the value to 0 and it recaps at 60 it's just fine. Any way to fix this?

Which roblox games are you experiencing this on? I've had this problem on Evade but setting my fps cap to around 200 fixes the problem.

**Comment by BokBoy** [comment]
> > When I uncap my fps, I don't get my full 144fps that I'd normally get. 
> 
> 
> 
> How is this remotely the same problem?

It's an issue with fps or ping

**Comment by CodingWithAnxiety** [comment]
> > > When I uncap my fps, I don't get my full 144fps that I'd normally get.
> > 
> > 
> > How is this remotely the same problem?
> 
> It's an issue with fps or ping

No, it's not, Bok. The issue you stated there is a performance issue, not a ping (network) issue. 

This issue should really be marked as closed imo.

**Comment by prodog666** [comment]
i have the same problem   i always got high ping on jailbreak,bloxfruits,bedwars,the stongest battleground games and more 
(not always high ping its like 90~100 then to 400 or more) pls help me ty
